src/electron.ts

```diff
@@ -36,23 +36,19 @@ app.on('ready', () => {
   win.setMenu(m);
 
   ipcMain.on('minimize', () => {
-    if (!current.isMinimized()) {
-      current.minimize();
-    }
+    current.minimize();
+  });
+
+  ipcMain.on('tabBaximize', () => {
+    current.isMaximized() ? current.unmaximize() : current.maximize();
   });
 
   ipcMain.on('maximize', () => {
-    let isFullScreen = null;
-    if (osPlatform === "linux" || osPlatform === "win32") {
-      isFullScreen = current.isMaximized();
-      if (!isFullScreen) {
-        current.maximize();
-      } else {
-       current.unmaximize()
-      }
+    let notMac = osPlatform === "linux" || osPlatform === "win32";
```

Comment:
This is ok, but it might be a little bit more advanced if you would write: `let notMac = ['linux', 'win32'].indexOf(osPlatform)!=-1`.